scripts/deploy.sh

```diff
@@ -1,48 +1,71 @@
 #!/bin/bash
-PROJECT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/..
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+PROJECT_ROOT=${SCRIPT_DIR}/..
 TEMPLATE_DIR="${PROJECT_ROOT}/templates"
 
-set -e
-
-# Based on https://gist.github.com/pkuczynski/8665367
-function parse_yaml() {
-    local prefix=$2
-    local s
-    local w
-    local fs
-    s='[[:space:]]*'
-    w='[a-zA-Z0-9_]*'
-    fs="$(echo @|tr @ '\034')"
-    sed -ne "s|^\($s\)\($w\)$s:$s\"\(.*\)\"$s\$|\1$fs\2$fs\3|p" \
-        -e "s|^\($s\)\($w\)$s[:-]$s\(.*\)$s\$|\1$fs\2$fs\3|p" "$1" |
-    awk -F"$fs" '{
-    indent = length($1)/2;
-    vname[indent] = $2;
-    for (i in vname) {if (i > indent) {delete vname[i]}}
-        if (length($3) > 0) {
-            vn=""; for (i=0; i<indent; i++) {vn=(vn)(vname[i])("_")}
-            printf("%s%s%s=(\"%s\")\n", "'"$prefix"'",vn, $2, $3);
-        }
-    }' | sed 's/_=/+=/g'
-}
+# from makefile
+BROKER_IMAGE=$1
+REGISTRY=$2
 
-function oc_create {
-    oc create -f $TEMPLATE_DIR/$@
-}
+# override with from my_local_dev_vars
+PROJECT="ansible-service-broker"
+OPENSHIFT_TARGET="https://kubernetes.default"
+OPENSHIFT_USER="admin"
```

Comment:
We should probably remove OPENSHIFT_USER and OPENSHIFT_PASS since they are no longer being used.

The template also needs to remove these, but that might be a different PR when we expose the path for token, target, and ca_file.